app/views/bookings/_current_bookings.html.erb

```diff
@@ -0,0 +1,34 @@
+
+<% count = 0 %>
+
+<% @bookings.each do |booking| %>
+  <% if booking.start_date < Date.parse(Time.now.strftime('%Y/%m/%d')) && booking.end_date > Date.parse(Time.now.strftime('%Y/%m/%d')) %>
```

Comment:
shouldn't it be <= ? because it's already current if it is today and also >= because it's still current if today is the last day?